buddypress/groups/single/members-loop.php

```diff
@@ -36,9 +36,12 @@
 
 	<?php
 	$count = groups_get_current_group()->total_member_count;
-	if ($count < 22) : ?>
+	global $bfc_dropdown_prefix;
+
+	if ($count <22) : ?>
```

Comment:
This looks like a magic number… What does it mean? Is there variable we could use for it? If not, maybe a comment would be helpful so we understand what it's for.

Reply:
I'll add a comment. The reasoning goes as follows: In parallel with limiting BFNow cohorts to 21 people, this limits the groups where members are listed with there self-intros to 21 people. Above 21, this members pages just shows avatars and names, like on the People directory page. Behind this choice of 21 is the sense that it's a good transition number between groups that feel reasonably intimate and those that feel "big".

Reply:
Excellent. And this is the only place for that breakpoint is reflected? If it does end up getting used elsewhere, it might be worth turning it into a variable that they all can use.

I realize I'm commenting on code that isn't actually being changed as part of this pull request, but it's nearby the change, so I happened to notice it :-)

Reply:
So far this is the only place. If we add more then it will certainly be worth converting to a variable. At some point it might also be good to convert it into a setting that could be accessed via the WP admin area but we're not ready for that yet.

Reply:
Cool. Making it a setting would be a big step, because then it's not under source control. Anyway, I support just commenting on it for the time being. Think of it as "appropriate generalization"… Not too little, but also not too much!